grammars/css.cson

```diff
@@ -329,183 +1298,348 @@
       }
       {
-        'match': '(?<![\\w-])(?i:arial|century|comic|courier|garamond|georgia|helvetica|impact|lucida|symbol|system|tahoma|times|trebuchet|utopia|verdana|webdings|sans-serif|serif|monospace)(?![\\w-])'
-        'name': 'support.constant.font-name.css'
+        'include': '#comment-block'
       }
       {
-        'include': '#numeric-values'
+        'include': '#escapes'
       }
       {
-        'include': '#color-values'
+        'include': '#functions'
       }
       {
-        'include': '#string-double'
+        'include': '#property-keywords'
       }
       {
-        'include': '#string-single'
+        'include': '#unicode-range'
       }
       {
-        'begin': '(rect)\\s*(\\()'
-        'beginCaptures':
-          '1':
-            'name': 'support.function.misc.css'
-          '2':
-            'name': 'punctuation.section.function.css'
-        'end': '(\\))'
-        'endCaptures':
-          '1':
-            'name': 'punctuation.section.function.css'
-        'patterns': [
-          {
-            'include': '#numeric-values'
-          }
-        ]
+        'include': '#numeric-values'
       }
       {
-        'begin': '(format|local|url|attr|counter|counters)\\s*(\\()'
-        'beginCaptures':
-          '1':
-            'name': 'support.function.misc.css'
-          '2':
-            'name': 'punctuation.section.function.css'
-        'end': '(\\))'
-        'endCaptures':
-          '1':
-            'name': 'punctuation.section.function.css'
-        'patterns': [
-          {
-            'include': '#string-single'
-          }
-          {
-            'include': '#string-double'
-          }
-          {
-            'match': '[^\'") \\t]+'
-            'name': 'variable.parameter.misc.css'
-          }
-        ]
+        'include': '#color-keywords'
       }
       {
-        'begin': '''(?x)
-          (?<![\\w-])
-          (
-            matrix(?:3d)?
-          | (?:translate|scale|rotate)(?:[XYZ]|3d)?
-          | skew[XY]?
-          | perspective
-          )
-          \\s*(\\()
-        '''
-        'beginCaptures':
-          '1':
-            'name': 'support.function.transform.css'
-          '2':
-            'name': 'punctuation.section.function.css'
-        'end': '\\)'
-        'endCaptures':
-          '0':
-            'name': 'punctuation.section.function.css'
-        'patterns': [
-          {
-            'include': '#numeric-values'
-          }
-          {
-            'match': ','
-            'name': 'punctuation.separator.css'
-          }
-        ]
+        'include': '#string'
       }
       {
-        'match': '\\!\\s*important'
+        'match': '!\\s*important(?![\\w-])'
         'name': 'keyword.other.important.css'
       }
     ]
+  'pseudo-classes':
+    'captures':
+      '1':
+        'name': 'punctuation.definition.entity.css'
+      '2':
+        'name': 'invalid.illegal.colon.css'
+    'match': '''(?xi)
+      (:)(:*)
```

Comment:
To extend angry red highlighting across every repeated colon. Unless for some reason only the second colon of `a::::::::first-child` should be marked as invalid...